modules/local/nf-core-modified/minimap2/align/main.nf

```diff
@@ -15,8 +15,10 @@ process MINIMAP2_ALIGN {
     tuple val(index_meta), path(index)
 
     output:
-    tuple val(meta), path("*.sam"), emit: sam
-    path "versions.yml"           , emit: versions
+    tuple val(meta), path("*.sam"),             emit: sam
+    tuple val(meta), path("*.sorted.bam"),      emit: sorted_bam
```

Comment:
sorry if I'm a broken record here and feel free to ignore this comment if I am, but just double checking that you need to actually output the sam here. Would be great to avoid if possible bc they're so darn big, but if I'm remembering correctly one of the down stream tools is dumb and you do need the sam file.

Reply:
edit: feel free to ignore this, confirmed racon needs sam

Reply:
yes I absolutely hate this requirement and I lost a good part of an afternoon messing around with this through github issues